x-pack/plugins/apm/server/lib/helpers/get_apm_alerts_client.ts

```diff
@@ -26,17 +28,18 @@ export async function getApmAlertsClient({
     throw Error('No alert indices exist for "apm"');
   }
 
-  type ApmAlertsClientSearchParams = Omit<
-    Parameters<typeof alertsClient.find>[0],
-    'index'
-  >;
+  type RequiredParams = ESSearchRequest & {
+    size: number;
```

Comment:
`track_total_hits` should also be required

```suggestion
    size: number;
    track_total_hits: boolean | number;
```

Reply:
ok, i removed it as we were not having it in our alerting queries. I will add it to our queries in that case.

Reply:
I see your comment now:

> We don't need the track_total_hits unless we also want to update our queries as currently we don't use it?

Yes, we do want to explicitly set it always. This provides a (small) perf improvement with minimal DX downside. If we don't need the count you can set it to `false` or `0` (I think they are equivalent).